cdap-hbase-spi/src/main/java/co/cask/cdap/spi/hbase/HBaseDDLExecutor.java

```diff
@@ -25,18 +25,21 @@
 import javax.annotation.Nullable;
 
 /**
- * Interface providing the HBase DDL operations.
+ * Interface providing the HBase DDL operations. Except the {@link #initialize} method,
+ * implementation of all other methods should be idempotent.
  */
 @Beta
 public interface HBaseDDLExecutor extends Closeable {
   /**
    * Initialize the {@link HBaseDDLExecutor}.
+   * This method gets called before any other DDL methods provided by this interface.
```

Comment:
this could be a little ambiguous. A little better to say, 'This method is called once when the executor is created, before any other methods are called.'